operator/pkg/reconciliation/handler.go

```diff
@@ -68,9 +69,10 @@ var log = logf.Log.WithName("reconciliation_handler")
 type ReconcileCassandraDatacenter struct {
 	// This client, initialized using mgr.client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
-	client   client.Client
-	scheme   *runtime.Scheme
-	recorder record.EventRecorder
+	client        client.Client
+	scheme        *runtime.Scheme
+	recorder      record.EventRecorder
+	SecretWatches dynamicwatch.DynamicWatches
```

Comment:
why is this field exported? can it not be?

Reply:
Yes it must be exported. It's used in package `cassandradatacenter` where we set up the watches in the controller.

Reply:
TODO: Add comment about why this is here.